tests/ro/creator/data/roc-02-emobon-observatory.yml

```
@@ -0,0 +1,71 @@
+uses: !strategy emobon-observatory  # directive to drive specific 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)